packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -320,6 +335,10 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   _videoOutput = [avFactory videoOutputWithPixelBufferAttributes:pixBuffAttributes];
   frameUpdater.videoOutput = _videoOutput;
 
+  _pixelBufferSynchronizationQueue =
+      dispatch_queue_create("io.flutter.video_player.pixelBufferSynchronizationQueue", NULL);
+  frameUpdater.pixelBufferSynchronizationQueue = _pixelBufferSynchronizationQueue;
```

Comment:
Is there a reason this is being created here rather than in the frame updater initialization?